### PR TITLE
925962 - Update the UG document for TreeMap.

### PR DESCRIPTION
**Pull request by sanjayvikramsf4373**
### Description: ###

I have updated the UG document for WPF treemap control.

### CODE REVIEW

File: wpf/TreeMap/Accessibility.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
```
Comment (Karthickmani97) [inline wpf/TreeMap/Accessibility.md:1]: @sanjayvikramsf4373 , change file name to `accessibility`

image

File: wpf/TreeMap/Accessibility.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+layout: post
+title: Accessibility in WPF TreeMap control | Syncfusion
+description: Learn here about Accessibility support with the Syncfusion WPF TreeMap (SfTreeMap) control, its elements and more.
+platform: wpf
+control: TreeMap
+documentation: ug
+---
+
+#  Accessibility Support in WPF TreeMap (SfTreeMap)
```
Comment (Karthickmani97) [inline wpf/TreeMap/Accessibility.md:10]: Accessibility support in WPF TreeMap (SfTreeMap)

File: wpf/TreeMap/Accessibility.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+The TreeMap is designed to efficiently interact with its elements and provides voice descriptions for its items.
```
Comment (Karthickmani97) [inline wpf/TreeMap/Accessibility.md:12]: The TreeMap is designed to efficiently interact with its elements, offering voice descriptions for each item in the treemap.

File: wpf/TreeMap/Accessibility.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+<table>
+<tr>
+<td>
+<b> Navigation Shortcut Keys </b> <br/><br/></td><td>
+<b> Descriptions </b> <br/><br/></td></tr>
+<tr>
+<td>
+Tab<br/><br/></td><td> 
+Moves the selection to the next item on the right in the TreeMap.
```
Comment (Karthickmani97) [inline wpf/TreeMap/Accessibility.md:27]: Moves the selection to the next item on the right side of the TreeMap.

File: wpf/TreeMap/Accessibility.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+<br/><br/></td></tr>
+<tr>
+<td>
+Shift + Tab<br/><br/></td><td>
+ Moves the selection to the previous item on the left in the TreeMap.
```
Comment (Karthickmani97) [inline wpf/TreeMap/Accessibility.md:32]: Moves the selection to the previous item on the left side of the TreeMap.

File: wpf/TreeMap/Accessibility.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+<br/><br/></td></tr>
+</table>
+
+N>When [SelectionMode] (https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.TreeMap.SfTreeMap.html#Syncfusion_UI_Xaml_TreeMap_SfTreeMap_SelectionModes) is set to Default, pressing a key clears the previous selection and selects the newly focused item. In contrast, when [SelectionMode] (https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.TreeMap.SfTreeMap.html#Syncfusion_UI_Xaml_TreeMap_SfTreeMap_SelectionModes) is set to Multiple, the previously selected items remain selected, and additional items are selected with each key press.
```
Comment (Karthickmani97) [inline wpf/TreeMap/Accessibility.md:36]: When [SelectionModes] (https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.TreeMap.SfTreeMap.html#Syncfusion_UI_Xaml_TreeMap_SfTreeMap_SelectionModes) is set to `Default`, each key press clears the previous focus and selects the new item. When [SelectionModes] (https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.TreeMap.SfTreeMap.html#Syncfusion_UI_Xaml_TreeMap_SfTreeMap_SelectionModes) is set to `Multiple`, the previously focused item remains selected while new items are selected with each key press.



Add API links for Default and Multiple Enum.

File: wpf/Image-Editor/Text.md [hunk 7]
```diff
@@ -52,6 +52,9 @@ You can make the text bold, italic, or underline by clicking the corresponding i
 
 Text can be aligned to the left, center, or right by clicking the corresponding icon. By default, text will be aligned at the left.
```
Comment (Karthickmani97) [inline wpf/Image-Editor/Text.md:54]: @sanjayvikramsf4373 , ImageEditor changes, share as separate PR, as facing CI issue

File: wpf/TreeMap/Accessibility.md (hunk 2 above)
Comment (sanjayvikramsf4373) [reply wpf/TreeMap/Accessibility.md]: Modified.

File: wpf/TreeMap/Accessibility.md (hunk 3 above)
Comment (sanjayvikramsf4373) [reply wpf/TreeMap/Accessibility.md]: Modified the content

File: wpf/TreeMap/Accessibility.md (hunk 4 above)
Comment (sanjayvikramsf4373) [reply wpf/TreeMap/Accessibility.md]: Modified the content.

File: wpf/TreeMap/Accessibility.md (hunk 5 above)
Comment (sanjayvikramsf4373) [reply wpf/TreeMap/Accessibility.md]: Modified the line.

File: wpf/TreeMap/Accessibility.md (hunk 6 above)
Comment (sanjayvikramsf4373) [reply wpf/TreeMap/Accessibility.md]: Added API links Default and Multiple enum.

File: wpf/Image-Editor/Text.md (hunk 7 above)
Comment (sanjayvikramsf4373) [reply wpf/Image-Editor/Text.md]: Reverted the changes.

**Review by Jeyasri-Murugan — APPROVED** [review]

**Pull request merged**